starknet_devnet/server.py

```diff
@@ -111,5 +111,18 @@ def handle(error: StarkException):
     """Handles the error and responds in JSON. """
     return {"message": error.message, "status_code": error.status_code}, error.status_code
 
+@app.route('/api', methods = ['GET'])
+def api():
+    """Print available endpoints."""
+    routes = {}
+    # pylint: disable=protected-access
+    for url in app.url_map._rules:
```

Comment:
Instead of disabling protected access linter rule, we could use `app.url_map.iter_rules()`